### PR TITLE
integrate coding.net

### PR DESCRIPTION
**Pull request by sennohub**

### CODE REVIEW

**Comment by sennohub** [comment]
Thank you for reply, this's my first pull request, so there are lots of things I do not know...and I had signed the CLA, and what should I do next?

File: go.mod [hunk 1]
```diff
@@ -1,8 +1,11 @@
-module github.com/drone/go-login
+module github.com/sennotech/go-login
```
Comment (tboerger) [inline go.mod]: That got to be reverted

File: login/bitbucket/bitbucket.go [hunk 2]
```diff
@@ -7,8 +7,8 @@ package bitbucket
 import (
 	"net/http"
 
-	"github.com/drone/go-login/login"
-	"github.com/drone/go-login/login/internal/oauth2"
+	"github.com/sennotech/go-login/login"
```
Comment (tboerger) [inline login/bitbucket/bitbucket.go]: Why have you changed all the import paths?

File: go.mod (hunk 1 above)
Comment (sennohub) [reply go.mod]: Sorry, this is an intermediate version of mine and I forgot to revert it. In addition, there is CI/CD in coding.net, so there is no need to integrate coding.net, and could you cancel the PR or need I cancel the PR...thank you for your reply.

Comment (tboerger) [reply go.mod]: There is also ci/cd in Gitlab, but we still support it ;)

**Pull request closed without merging**